Content.Client/Graphics/Overlays/SingularityOverlay.cs

```diff
@@ -62,29 +62,30 @@ protected override void Draw(DrawingHandleBase handle, OverlaySpace currentSpace
 
 
 
-        //Queries all singulos on the map and either adds or removes them from the list of singulos to render depending on their location and existence.
+        //Queries all singulos on the map and either adds or removes them from the list of rendered singulos based on whether they should be drawn (in range? on the same z-level/map? singulo entity still exists?)
         private float _maxDist = 15.0f;
         private void SingularityQuery()
         {
             var currentEyeLoc = _eyeManager.CurrentEye.Position;
+            var currentMap = _eyeManager.CurrentMap; //TODO: support multiple viewports once it is added
 
             var singuloComponents = _componentManager.EntityQuery<IClientSingularityInstance>();
```

Comment:
This ignores paused singulos so not sure if you want the shader still going if the singulo is paused.